src/west/main.py

```diff
@@ -26,9 +26,16 @@
 from west.commands import external_commands, \
     CommandError, CommandContextError, ExtensionCommandError
 from west.commands.project import List, ManifestCommand, Diff, Status, \
-    SelfUpdate, ForAll, WestUpdated, PostInit, Update
+    SelfUpdate, ForAll, PostInit, Update
+from west.commands.common import Help
 from west.manifest import Manifest, MalformedConfig
-from west.util import quote_sh_list
+from west.util import quote_sh_list, WestReload
```

Comment:
I kind of feel like WestReload belongs in `west.commands`, since commands should be the only things that can raise the exception. Not married to the idea, though. What do you think?

Reply:
I am indifferent in this case. I guess it could be raised by main itself in the future but I am fine with either

Reply:
Let's keep it here for now until we're ready to refactor to commit to a real API